panoptes_aggregation/tests/extractor_tests/test_question_extractor.py

```diff
@@ -87,3 +87,29 @@
     kwargs={'task': 'T1'},
     test_name='TestSingleTwoTasksKey'
 )
+
+single_pluck_classification = {"id": 359841171,
+    "annotations": [{
+        "task": "T0",
+        "value": "0"
+    }],
+    "subject": {
+        "id": 68561182,
+        "metadata": {
+            "filename": "uds_H.gf19767.png",
+            "uber_flag_digit": "4",
+            "is_gold_standard": "False"
+        }
+    }
+}
+
+single_pluck_keys = '{"gold_standard": "subject.metadata.is_gold_standard", "true_value": "subject.metadata.uber_flag_digit"}'
```

Comment:
As noted above, this should be a dictionary rather than a string at this point.  Minor note, to keep the tests a bit more readable (and future git-diffs easier to see changes) I tend to space all dicts out to one item per line:

```python
single_pluck_keys = {
    "gold_standard": "subject.metadata.is_gold_standard",
    "true_value": "subject.metadata.uber_flag_digit"
}
```

Reply:
I might be misunderstanding your point, but wouldn't it still be a string at this point, because the `pluck` input parameter will be evaluated as a string in the `extractor_wrapper`? So, if I'm understanding correctly, in the test cases, the `pluck` value would be the raw string from the URL parameters.